package.json

```diff
@@ -4,16 +4,29 @@
   "private": true,
   "dependencies": {
     "@material-ui/core": "^4.5.1",
+    "@material-ui/icons": "^4.5.1",
     "@material-ui/styles": "^4.5.0",
     "react": "^16.8.3",
     "react-dom": "^16.8.3",
     "react-router-dom": "^5.1.2",
-    "react-scripts": "1.1.5"
+    "react-scripts": "3.3.0"
```

Comment:
`npm install` failed for me with this `react-scripts` version for some reason, saying that version isn't available. And the [package](https://www.npmjs.com/package/react-scripts) shows 3.2.0 as the latest install-able version, which worked for me.